vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/WaitSteps.java

```diff
@@ -265,7 +266,7 @@ public void waitTillElementIsStale(Locator locator)
     @When("I wait until element located `$locator` contains text '$text'")
     public void waitTillElementContainsText(Locator locator, String text)
     {
-        waitActions.wait(getSearchContext(),
+        waitActions.wait(uiContext.getSearchContext(),
```

Comment:
maybe uiContext could be a part of wait actions and we remote the first parameter?